apps/rotm/behaviours/remove-image.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+module.exports = superclass => class extends superclass {
+
+  getValues(req, res, callback) {
+    if (req.get('Referrer').includes('/image')) {
+      req.sessionModel.unset('image');
+      req.sessionModel.unset('image-preview');
+      req.sessionModel.unset('image-url');
+    }
```

Comment:
good spot, I'll investigate

Reply:
when you say model, do you mean the sessionModel at the point this behaviour is triggered? I've looked in the sessionModel and I can't find any values regarding the `data` or `mimetype`

Reply:
yes the sessionModel

Reply:
OK, I just want to be sure that users won't end up with the wrong data (image) being saved

Reply:
Cool. However, perhaps, I'm not searching for it properly.

When does it get saved into the session Model from the model? the first time in saveValues, would the data be saved to the sessionmodel? I inspected that in the behaviour saveImage and I couldn't find it.  I could find it in the model but not the sessionModel. Would it be found by doing req.sessionModel.get('data')

Reply:
Yes, if a property has been set on the `sessionModel` it can be accessed with `req.sessionModel.get('data')`. 
If that gives you nothing, then there is not property called `data` on the `sessionModel`.

FYI, it never gets saved to the `sessionModel`, I know that because I followed the code to the `UploadModel`, which `POST`'s the attributes to the S3 API, but it doesn't `set to the attributes to `form.values`... etc and delegate to `saveValues` or explicitly set to `sessionModel`

Reply:
👍